File: Lecture 4/lecture4_src/phonebook0.c
// Реализация телефонного справочника без использования структуры

#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    // Объявление и инициализация массивов строк
    string names[] = {"Carter", "David", "John"};
    string numbers[] = {"+1-617-495-1000", "+1-617-495-1000", "+1-949-468-2750"};

    // Поиск по имени, введённому пользователем
    string name = get_string("Name: ");
    for (int i = 0; i < 3; i++)
    {
        if (strcmp(names[i], name) == 0)
        {
            printf("Found %s\n", numbers[i]);
            return 0;
        }
    }
    printf("Not found\n");
    return 1;
}

File: Lecture 7/lecture7_src/3/phonebook0.py
# Реализация линейного поиска имён с использованием цикла for

# Объявление списка names
names = ["Carter", "David", "John"]

# Запрос имён
name = input("Name: ")

# Поиск по имени
for n in names:
    if name == n:
        print("Found")
        break
else:
    print("Not found")
